README.md

```diff
@@ -1,41 +1,49 @@
 # BastCastle Simple Home Automation using Home Assistant
 
 ## The haHA Gateway Drug
-As a part of a cloud computing practicum, a team of six came together to build a home automation process using Home Assitant for three specific use cases: locks, curtains, and vents. We started testing with lights, so we kept those scipts in for funzies. Further, we built our process in the cloud but optimized it for migration to a PI-Hole. 
+As a part of a cloud computing practicum, a team of six came together to build a home automation
+process using Home Assistant for three specific use cases: locks, curtains, and vents. We started testing
+with lights, so we kept those scripts in for fun. Further, we built our process in the cloud within an AWS
+EC2 instance but optimized it for migration to a Pi-hole.
 
-Take our blood, sweat, and tears and build upon this foundation. Add addtional use cases of your own. At this point, in the world of home automation IoT, the sky is the limit: lights, locks, solar panels, televisions, alarams, cameras, vents, curtains, blinds, switches, sprinklers, etc. 
+Take our blood, sweat, and tears and build upon this foundation. Add additional use cases of your own.
+At this point, in the world of home automation IoT, the sky is the limit: lights, locks, solar panels,
+televisions, alarms, cameras, vents, curtains, blinds, switches, sprinklers, etc.
 
 WARNING: once you get started with haHA, you won't stop! 
 
-### Contributing Developers (should we link these to LI profiles?)
-- Alexi Most
-- Lucas Knezevich
-- Jason Beutler
-- Victor Navarro
-- Ashray Thapa
-- Brandon Dombrowsky
+### Contributing Developers
+- [Alexi Most](https://www.linkedin.com/in/alyoshenka/) 
+- [Lucas Knezevich](https://www.linkedin.com/in/lucasknezevich/)
+- [Jason Beutler](https://www.linkedin.com/in/jasonpbeutler/)
+- Victor Navarro (need his link)
+- [Ashray Thapa](https://www.linkedin.com/in/ashray-thapa/)
+- [Brandon Dombrowsky](https://www.linkedin.com/in/brandondombrowsky/)
 
 ## Getting Started
-This code is compatible with... AWS/yaml based scripting/etc
-
-## Cloud vs PI-Hole
-- blah-blah-blah
-- blah-blah-blah
+Though most configuration sources found online highly recommend running HA locally on a Pi-hole, our
+assignment was to deploy to a cloud service. As such, our CI/CD implementation is optimized for the
+cloud and automatically deploying repository changes to our AWS EC2 instance.
 
 ## Installation 
-    1. Do this 
-    2. Then do this
-    3. Then do this
-    4. Then do this
-    5. Finally, you're done
-    
-## Checkout the [Wiki page] (https://github.com/brandondombrowsky/BastCastle/wiki)
+1. Install Home Assistant locally. Though there are many ways to do this, the one I found the simplest for
```

Comment:
Yes this can be done, but it's not absolutely necessary because you can just go to the EC2 link. Also, feel free to add a link to the wiki page that has the HA docker setup, preliminary steps are on the Wireguard page.